docs/pages/includes/role-spec.mdx

```diff
@@ -97,8 +97,10 @@ spec:
        mode: extension
        name: login@github.com
        value: "{{ external.github_login }}"
-    # Controls whether this role supports auto provisioning of users.
+    # Controls whether this role supports auto provisioning of SSH users.
     create_host_user: true
+    # Controls whether this role requires automatic database user provisioning.
```

Comment:
Comparing this to the previous `create` example, should this be "supports" instead of "requires"?